webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -159,6 +159,10 @@ trait NodeInfoService {
    */
   def getAllSystemNodeIds() : Box[Seq[NodeId]]
 
+  /**
+   * Remove a node from cache. It cannot fail - if node is not in cache, it is a success
+   */
+  def removeNodeFromCache(nodeId: NodeId): IOResult[Unit]
```

Comment:
it should only be in `NodeInfoServiceCached` version, or in a separated trait. It has nothing to do with the main service interface

Reply:
indeed